app-backup/amanda/amanda-3.5.ebuild

```diff
@@ -0,0 +1,479 @@
+	insopts -m0640
+	# Installing Amanda Xinetd Services Definition
+	einfo "Installing xinetd service file"
+	insinto /etc/xinetd.d
+	if use minimal ; then
+		newins "${MYFILESDIR}"/amanda-xinetd-2.6.1_p1-client amanda
+	else
+		newins "${MYFILESDIR}"/amanda-xinetd-2.6.1_p1-server amanda
+	fi
+
+	if ! use minimal; then
+		einfo "Installing Sample Daily Cron Job for Amanda"
+		insinto /etc/cron.daily
+		newins "${MYFILESDIR}/amanda-cron" amanda
+	fi
+
+	einfo "Installing systemd service and socket files for Amanda"
+	systemd_dounit "${FILESDIR}"/amanda.socket || die
+	systemd_newunit "${FILESDIR}"/amanda.service 'amanda@.service' || die
+
+	insinto /etc/amanda
+	einfo "Installing .amandahosts File for ${AMANDA_USER_NAME} user"
+	doins "${T}/amandahosts"
+	fperms 600 /etc/amanda/amandahosts
+
+	dosym /etc/amanda/amandahosts "${AMANDA_USER_HOMEDIR}/.amandahosts"
+	insinto "${AMANDA_USER_HOMEDIR}"
+	einfo "Installing .profile for ${AMANDA_USER_NAME} user"
+	newins "${MYFILESDIR}/amanda-profile" .profile
+
+	insinto /etc/amanda
+	doins "${S}/example/amanda-client.conf"
+	if ! use minimal ; then
+		insinto "/etc/amanda/${AMANDA_CONFIG_NAME}"
+		doins "${S}/example/amanda.conf"
+		doins "${S}/example/disklist"
+		keepdir "${AMANDA_USER_HOMEDIR}/${AMANDA_CONFIG_NAME}/index"
+	fi
+
+	keepdir "${AMANDA_TAR_LISTDIR}"
+	keepdir "${AMANDA_USER_HOMEDIR}/amanda"
+	keepdir "${AMANDA_TMPDIR}/dumps"
+	# Just make sure it exists for XFS to work...
+	use xfs && keepdir /var/xfsdump/inventory
+
+	local i
+	for i in "${AMANDA_USER_HOMEDIR}" "${AMANDA_TAR_LISTDIR}" \
+		"${AMANDA_TMPDIR}" /etc/amanda; do
+		einfo "Securing directory (${i})"
+		fowners -R ${AMANDA_USER_NAME}:${AMANDA_GROUP_NAME} ${i}
+	done
+	# Do NOT use -R
+	fperms 0700 \
+		"${AMANDA_USER_HOMEDIR}" "${AMANDA_TAR_LISTDIR}" \
+		"${AMANDA_TMPDIR}" "${AMANDA_TMPDIR}/dumps" \
+		 "${AMANDA_USER_HOMEDIR}/amanda" \
+		 /etc/amanda
+
+	if ! use minimal ; then
+		fperms 0700 \
+			 "${AMANDA_USER_HOMEDIR}/${AMANDA_CONFIG_NAME}" \
+	         /etc/amanda/${AMANDA_CONFIG_NAME}
+	fi
+
+	einfo "Setting setuid permissions"
+	amanda_permissions_fix "${D}"
+
+	# Relax permissions again
+	insopts -m0644
+
+	# docs
+	einfo "Installing documentation"
+	dodoc AUTHORS ChangeLog DEVELOPING NEWS README ReleaseNotes UPGRADING
+	# our inetd sample
+	einfo "Installing standard inetd sample"
+	newdoc "${MYFILESDIR}/amanda-inetd.amanda.sample-2.6.0_p2-r2" amanda-inetd.amanda.sample
+	# Amanda example configs
+	einfo "Installing example configurations"
+	rm "${D}"/usr/share/amanda/{COPYRIGHT,ChangeLog,NEWS,ReleaseNotes}
+	mv "${D}/usr/share/amanda/example" "${D}/usr/share/doc/${PF}/"
+	docinto example1
+	newdoc "${FILESDIR}/example_amanda.conf" amanda.conf
+	newdoc "${FILESDIR}/example_disklist-2.5.1_p3-r1" disklist
+	newdoc "${FILESDIR}/example_global.conf" global.conf
+
+	einfo "Cleaning up dud .la files"
+	perl_set_version
+	find "${D}"/"${VENDOR_LIB}" -name '*.la' -print0 |xargs -0 rm -f
```

Comment:
This line should end with a `|| die` call.

Reply:
The `|| die` will give you false positives if there are no `*.la` files, you also need --no-run-if-empty.
I suggest this as a overall improvement:
`find "${D}"/"${VENDOR_LIB}" -name '*.la' -delete || die`
If there are no la files, it will succeed. If there is a file, and it cannot delete it, it will trigger the die.